src/FixedDataTableRow.js

```diff
@@ -177,6 +197,11 @@ class FixedDataTableRowImpl extends React.Component {
           {scrollableColumns}
           {columnsLeftShadow}
         </div>
+        {rowExpanded && <div
```

Comment:
Thanks for moving the logic into here.  Now can we delete this wrapping div and just render the rowExpanded div directly?